src/pages/en/developers/apis/4-realtime-api/vehicle-positions-2/index.md

```diff
@@ -0,0 +1,409 @@
+| `drst`    | Integer               |                                      |Door status.<br/>`0` if all the doors are closed.<br/>`1` if any of the doors are open.       
+| `oday`    | String                |  `da`, `dout`                        | Operating day of the trip. The exact time when an operating day ends depends on the route. For most routes, the operating day ends at 4:30 AM on the next day. In that case, for example, the final moment of the operating day `"2018-04-05"` would be at 2018-04-06T04:30 local time.
+| `jrn`     | Integer               |  `da`, `dout`, `ba`, `bout`          | Internal journey descriptor, not meant to be useful for external use. 
+| `line`    | Integer               |  `da`, `dout`, `ba`, `bout`          | Internal line descriptor, not meant to be useful for external use. 
+| `start`   | String                |  `da`, `dout`, `ba`, `bout`          | Scheduled start time of the trip, i.e. the scheduled departure time from the first stop of the trip. The format follows `HH:mm` in 24-hour local time, not the 30-hour overlapping operating days present in GTFS. Matches `start_time` in the topic.
+| `loc`     | String                |                                      | Location source, either `GPS`, `ODO`, `MAN` or `N/A`. <ul><li>`GPS` - location is received from GPS</li><li>`ODO` - location is calculated based on odometer value</li><li>`MAN` - location is specified manually</li><li>`N/A` - location is unavailable</li></ul>
+| `stop`    | String                | `da`, `dout`, `ba`, `bout`           | ID of the stop related to the event (e.g. ID of the stop where the vehicle departed from in case of `dep` event or the stop where the vehicle currently is in case of `vp` event).<br/>`null` if the event is not related to any stop.
+| `route`   | String                | `da`, `dout`, `ba`, `bout`           | ID of the route the vehicle is currently running on. Matches `route_id` in the topic.
+| `occu`    | Integer               | `da`, `dout`, `ba`, `bout`           | Integer describing passenger occupancy level of the vehicle. Valid values are on interval `[0, 100]`. However, currently only values used are `0` *(= vehicle has space and is accepting passengers)* and `100` *(= vehicle is full and might not accept passengers)*
+| `ttarr`   | String                | `vp`, `da`, `dout`, `ba`, `bout`, `vja`, `vjout` | UTC timestamp of scheduled arrival time to the stop
+| `ttdep`   | String                | `vp`, `da`, `dout`, `ba`, `bout`, `vja`, `vjout` | UTC timestamp of scheduled departure time from the stop
+| `dr-type` | Integer               | Other than `da`, `dout`, `ba`, `bout`, `vja`, `vjout` | Type of the driver, either `0` or `1`. <ul><li>`0` = normal driver</li><li>`1` = service technician</li></ul>
```

Comment:
Is this the correct way? According to docs:
"Driver Sign-in type (1 = driver, 0 = service technician)"

Reply:
Fixed